doc/api/magic-plot.md

```diff
@@ -160,3 +160,100 @@ ax = %sqlplot histogram --table no_nulls --column body_mass_g --with no_nulls
 ax.set_title("Body mass (grams)")
 _ = ax.grid()
 ```
+## `%sqlplot bar`
+
+Shortcut: `%sqlplot bar`
```

Comment:
there is some print statement when using `bar`

![image](https://github.com/ploomber/jupysql/assets/989250/41896085-7639-4516-a8d4-4f02d57d5877)

Reply:
we should remove it